### PR TITLE
Update to 616fee

### PR DESCRIPTION
**Pull request by frostedcmos**
Signed-off-by: Alexander Oganezov <alexander.a.oganezov@intel.com>

### CODE REVIEW

File: f0b9f992793be46f1c6ae47b30d1c3ccb525cfbf...616fee16a3c09eb93d804b627e64ea1c84c6b4d4.patch [hunk 1]
```diff
@@ -0,0 +1,484 @@
...
+@@ -2,10 +2,10 @@
+ 
+ BMI_VERSION=master
+ CMAKE_VERSION_MAJOR=3.15
+-CMAKE_VERSION_MINOR=1
++CMAKE_VERSION_MINOR=3
+ MPI_VERSION=3.3.1
+ CCI_VERSION=2.1
+-OFI_VERSION=1.8.0
++OFI_VERSION=1.8.1
```
Comment (brianjmurrell) [inline f0b9f992793be46f1c6ae47b30d1c3ccb525cfbf...616fee16a3c09eb93d804b627e64ea1c84c6b4d4.patch]: Is this going to need a libfabric update also then?

Comment (frostedcmos) [reply f0b9f992793be46f1c6ae47b30d1c3ccb525cfbf...616fee16a3c09eb93d804b627e64ea1c84c6b4d4.patch]: We are already past that version in cart itself, my local cart master shows as:
fi_info: 1.9.0a1

Comment (brianjmurrell) [reply f0b9f992793be46f1c6ae47b30d1c3ccb525cfbf...616fee16a3c09eb93d804b627e64ea1c84c6b4d4.patch]: But is the RPM there yet?

File: mercury.spec [hunk 2]
```diff
@@ -10,6 +10,7 @@ URL:		http://mercury-hpc.github.io/documentation/
 Source0:	https://github.com/mercury-hpc/%{name}/releases/download/v%{version}/%{name}-%{version}.tar.bz2
 Patch1:		https://github.com/mercury-hpc/mercury/compare/v1.0.1...cc0807e8377e129945834d292be21a6667a8cbb3.patch
 Patch2:		https://github.com/mercury-hpc/mercury/compare/cc0807e8377e129945834d292be21a6667a8cbb3...f0b9f992793be46f1c6ae47b30d1c3ccb525cfbf.patch
+Patch3:		https://github.com/mercury-hpc/mercury/compare/f0b9f992793be46f1c6ae47b30d1c3ccb525cfbf...616fee16a3c09eb93d804b627e64ea1c84c6b4d4.patch
```
Comment (brianjmurrell) [inline mercury.spec]: This patch needs adding to the `PATCHES` variable in `Makefile`.  It's an open TODO to have the Makefile extract that info automatically, but just not done yet.

File: f0b9f992793be46f1c6ae47b30d1c3ccb525cfbf...616fee16a3c09eb93d804b627e64ea1c84c6b4d4.patch (hunk 1 above)
Comment (brianjmurrell) [reply f0b9f992793be46f1c6ae47b30d1c3ccb525cfbf...616fee16a3c09eb93d804b627e64ea1c84c6b4d4.patch]: Ahhh.  Seems libfabric RPM is current to CaRT:
https://github.com/daos-stack/libfabric/blob/a2b3e756c048f53a853973098d6944555e0784f3/libfabric.spec#L16

**Pull request merged**